src/main/kotlin/com/example/goodtripserver/trip/service/TripService.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.goodtripserver.trip.service
+
+import com.example.goodtripserver.trip.model.AddCountryRequest
+import com.example.goodtripserver.trip.model.AddNoteRequest
+import com.example.goodtripserver.trip.model.AddTripRequest
+import com.goodtrip.goodtripserver.database.models.Trip
+import org.springframework.http.ResponseEntity
+
+interface TripService {
```

Comment:
Service should not wrap any data additionaly (in ResponseEnitty), Service just processes the real data into something different that makes sense. Service can be used not only for processing rest, but in other parts of application, basically I would move all the data wrapping into ResponseEntity to the Controller itself